fe/fe-core/src/main/java/com/starrocks/catalog/Column.java

```diff
@@ -464,6 +469,22 @@ public String getCalculatedDefaultValue() {
         return null;
     }
 
+    // if the column have a default value or default expr can be calculated like now(). return calculated value
+    // else for a batch of every row different like uuid(). return null
+    // require specify currentTimestamp. this will get the default value of the incoming time
+    // base on the incoming time
+    public String getCalculatedDefaultValueWithTime(long currentTimestamp) {
```

Comment:
This function is only used to a const default value like "-1" or now().
If the default value is uuid(), this function is not suitable.
```
calculateDefaultValue()
```